packages/monaco/src/browser/monaco-editor-content-menu.ts

```diff
@@ -70,42 +70,44 @@ export class MonacoEditorContentMenuContribution implements FrontendApplicationC
         );
         return ObservableUtils.autorunWithDisposables(({ toDispose }) => {
             const menuNodes = menuNodesObservable.get();
-            const firstMatchObservable = ObservableFromEvent.create(contextKeyService.onDidChangeContext, () => this.withContext(context,
-                () => menuNodes.find(menuNode => menuNode.isVisible(EDITOR_CONTENT_MENU, this.contextKeyService, undefined, editorWidget))
-            ));
-            // eslint-disable-next-line @typescript-eslint/no-shadow
-            toDispose.push(ObservableUtils.autorunWithDisposables(({ toDispose }) => {
-                const firstMatch = firstMatchObservable.get();
-                if (firstMatch) {
-                    const button = new MonacoEditorOverlayButton(editor, firstMatch.label);
-                    toDispose.push(button);
-                    toDispose.push(button.onClick(() =>
-                        this.withContext(context, () => firstMatch.run(EDITOR_CONTENT_MENU, editorWidget))
-                    ));
+            if (menuNodes) {
```

Comment:
[nitpick] Although this additional check will certainly not hurt anything, it does not look like it's really necessary, as `menuNodes` is `CommandMenu[]` and cannot be falsy here.